src/server/common/model/answer/checkbox/checkbox.test.js

```diff
@@ -0,0 +1,243 @@
+import { CheckboxAnswer } from './checkbox.js'
+
+/** @import {CheckboxConfig} from './checkbox.js' */
+
+const checkboxEmptyError = 'Select at least one checkbox'
+
+/** @type {CheckboxConfig} */
```

Comment:
We could do it more generic and not use the badger page values, but I can actually do that once I've worked on that page